mxcube3/routes/Login.py

```diff
@@ -101,6 +101,10 @@ def signout():
     mxcube.queue = qutils.new_queue()
     mxcube.shapes.clear_all()
 
+    if mxcube.CURRENTLY_MOUNTED_SAMPLE_IS_MANUAL:
```

Comment:
Do we really need this check cant we just check id the sample changer is available and have a sample mounted
if its not the case its manual ?